### PR TITLE
[zh-cn]sync service.md

### PR DESCRIPTION
**Pull request by asa3311**
[zh-cn]sync service.md

### CODE REVIEW

**Comment by windsonsea** [comment]
Please include this change too before line 235:

image

**Comment by asa3311** [comment]
done

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -286,14 +290,14 @@ The default protocol for Services is
...
-port definitions on a Service object.
+Because many Services need to expose more than one port, Kubernetes supports
++[multiple port definitions](#multi-port-services) for a single Service.
 Each port definition can have the same `protocol`, or a different one.
 -->
 服务的默认协议是 [TCP](/zh-cn/docs/reference/networking/service-protocols/#protocol-tcp)；
 你还可以使用任何其他[受支持的协议](/zh-cn/docs/reference/networking/service-protocols/)。
 
-由于许多服务需要公开多个端口，因此 Kubernetes 在服务对象上支持多个端口定义。
+由于许多服务需要公开多个端口，因此 Kubernetes 在服务对象上支持多个端口定义 [多端口定义](#multi-port-services)。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/service.md:300]:
```suggestion
由于许多服务需要公开多个端口，所以 Kubernetes 针对单个服务支持 [多个端口定义](#multi-port-services)。
```

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**